requirements_mtj.txt

```diff
@@ -2,9 +2,10 @@ torch >= 1.9, < 1.13
 numpy
 tqdm
 requests
-dm-haiku == 0.0.5
-jax == 0.2.21
-jaxlib >= 0.1.69, <= 0.3.7
+dm-haiku==0.0.9
+jax==0.3.25
+jaxlib==0.3.25
+transformers == 4.28.0
```

Comment:
Transformers 4.28.0 has a different calling syntax for generation_logits_process so in [aiserver.py#2000](../../blob/main/aiserver.py#L2000) would need something like this, or the main requirements.txt would have to be updated as well, and aiserver.py changed to use `transformers.generation.logits_process` in place of `transformers.generation_logits_process`

```
    if transformers.__version__ >= "4.28.0":
        from transformers.generation import logits_process as generation_logits_process
    else:
        import transformers.generation_logits_process as generation_logits_process
    # Allow bad words filter to ban <|endoftext|> token
    def new_init(self, bad_words_ids: List[List[int]], eos_token_id: int):
        return new_init.old_init(self, bad_words_ids, -1)
    new_init.old_init = generation_logits_process.NoBadWordsLogitsProcessor.__init__
    generation_logits_process.NoBadWordsLogitsProcessor.__init__ = new_init
```